generators/chipyard/src/main/scala/config/AbstractConfig.scala

```diff
@@ -61,6 +63,7 @@ class AbstractConfig extends Config(
   new chipyard.config.WithFrontBusFrequency(500.0) ++               // Default 500 MHz fbus
   new chipyard.config.WithOffchipBusFrequency(500.0) ++             // Default 500 MHz obus
 
+  new testchipip.soc.WithChipIdPin ++                               // add a chip id pin for setting chip id in multi-chip configs
```

Comment:
Can we leave out the chipid pin in default designs for now?

The IOBinders/Harnessbinders should stay, but the pin should not be enabled.